template/typescript/src/cli.tsx

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+import React from 'react';
+import {render} from 'ink';
+import meow from 'meow';
+import {App} from './ui';
+
+const cli = meow(`
+	Usage
+	  $ %NAME%
+
+	Options
+		--name  Your name
+
+	Examples
+	  $ %NAME% --name=Jane
+	  Hello, Jane
+`);
+
+render(React.createElement(App, cli.flags));
```

Comment:
I'm not really sure about this but I have to pass `cli.flags` to `render` instead of `React.createElement` to make this works in Typescript.

```suggestion
render(React.createElement(App), cli.flags);
```